js/apps/system/_admin/aardvark/APP/react/src/views/views/Components/AccordionExclusive.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import 'semantic-ui-css/semantic.min.css';
+import { Accordion, Form } from "semantic-ui-react";
+
+const AccordionExclusive = ({panels}) => (
+  <Accordion
+    defaultActiveIndex={[0, 2]}
+    panels={panels}
+    exclusive={false}
```

Comment:
Shouldn't this be `true`? At least that's what the name implies.

Reply:
This file is obsolete so I deleted it.